confignode/src/main/java/org/apache/iotdb/confignode/consensus/statemachine/PartitionRegionStateMachine.java

```diff
@@ -141,13 +141,15 @@ public void notifyLeaderChanged(ConsensusGroupId groupId, TEndPoint newLeader) {
     if (currentNode.equals(newLeader)) {
       LOGGER.info("Current node {} becomes Leader", newLeader);
       configManager.getProcedureManager().shiftExecutor(true);
-      configManager.getLoadManager().start();
+      configManager.getLoadManager().startLoadBalancingService();
+      configManager.getNodeManager().startHeartbeatService();
```

Comment:
What about moving `startHeartbeatService` in front of `startLoadBalancingService`

Reply:
I'm not sure either...